packages/genesys-spark-components/src/components/beta/gux-side-panel/example.html

```diff
@@ -0,0 +1,140 @@
+<div id="side-panel-wrapper">
+  <div id="content">
+    <h1>Side Panel</h1>
+
+    <p>
+      You can listen to the 'sidePanelDismiss' event on the
+      `gux-side-panel-beta` element to know when the side panel is to be
+      hidden/removed.
+    </p>
+
+    <gux-button accent="secondary" id="toggle-small">Toggle small</gux-button>
+    <gux-button accent="secondary" id="toggle-medium">Toggle medium</gux-button>
+
+    <h2>Modal Side Panel</h2>
+
+    <p>You can show/hide the `gux-modal-side-panel` via two mechanisms:</p>
+    <ol>
+      <li>
+        Use the `showModal()` and `close()` methods on a `gux-modal-side-panel`
+        element
+      </li>
+      <li>Set the `open` property or attribute on `gux-modal-side-panel`</li>
+    </ol>
+
+    <gux-modal-side-panel-beta id="modal-example">
+      <gux-side-panel-heading
+        slot="heading"
+        level="3"
+        icon="fa/diamond-regular"
+      >
+        Side panel title
+      </gux-side-panel-heading>
```

Comment:
I would suggest the folder structure for `gux-side-panel` to be something like this:

- `gux-side-panel`

  - `gux-side-panel-heading`
  - `gux-modal-side-panel`